obspy/imaging/util.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+"""
+Waveform plotting utilities.
+
+:copyright:
+    The ObsPy Development Team (devs@obspy.org)
+:license:
+    GNU General Public License (GPL)
```

Comment:
Thanks, that was before the License change and I must have overlooked it during updating to current master..